packages/help-center/src/components/help-center-article.tsx

```diff
@@ -64,6 +64,37 @@ export const HelpCenterArticle = () => {
 		}
 	}, [ post, query, sectionName ] );
 
+	// Trigger event for each section scrolled into view
+	useEffect( () => {
+		if ( elementRef.current ) {
+			const observer = new IntersectionObserver(
+				( entries ) => {
+					entries.forEach( ( entry ) => {
+						if ( entry.isIntersecting ) {
+							const tracksData = {
+								force_site_id: true,
+								location: 'help-center',
+								post_url: post?.URL,
+								post_id: post?.ID,
+								blog_id: post?.site_ID,
+								section_id: entry?.target?.id,
+							};
+							recordTracksEvent( 'calypso_helpcenter_article_section_scroll_viewed', tracksData );
```

Comment:
On our marketing pages, we have the event `wpcom_page_section_view`, maybe we can use `calypso_helpcenter_article_section_view`